awx/ui/client/src/instance-groups/instance-groups.route.js

```diff
@@ -14,7 +14,7 @@ export default {
                 page_size: '20',
                 order_by: 'name'
             },
-            dynamic: true
+            dyanmic: true
```

Comment:
I see `dynamic` (correct spelling) changed to `dyanmic` (incorrect spelling) in several places -- does this mean that leaving `dynamic` unset/undefined leads to the correct default behavior from UI router? If so, we should remove these lines.